tests/vector/test_step_compatibility_vector.py

```diff
@@ -1,88 +0,0 @@
-import numpy as np
-import pytest
-
-import gym
-from gym.spaces import Discrete
-from gym.vector import AsyncVectorEnv, SyncVectorEnv
-
-
-class OldStepEnv(gym.Env):
-    def __init__(self):
-        self.action_space = Discrete(2)
-        self.observation_space = Discrete(2)
-
-    def reset(self):
-        return 0, {}
-
-    def step(self, action):
-        obs = self.observation_space.sample()
-        rew = 0
-        done = False
-        info = {}
-        return obs, rew, done, info
-
-
-class NewStepEnv(gym.Env):
-    def __init__(self):
-        self.action_space = Discrete(2)
-        self.observation_space = Discrete(2)
-
-    def reset(self):
-        return 0, {}
-
-    def step(self, action):
-        obs = self.observation_space.sample()
-        rew = 0
-        terminated = False
-        truncated = False
-        info = {}
-        return obs, rew, terminated, truncated, info
-
-
-@pytest.mark.parametrize("VecEnv", [AsyncVectorEnv, SyncVectorEnv])
-def test_vector_step_compatibility_new_env(VecEnv):
```

Comment:
Do we not want to test this anymore?

Reply:
This was purely for vector environments. These tests should be covered in the step api testing